src/features/unified-share-modal/UnifiedShareForm.js

```diff
@@ -0,0 +1,577 @@
+// @flow
+
+import * as React from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import LoadingIndicatorWrapper from '../../components/loading-indicator/LoadingIndicatorWrapper';
+import { Link } from '../../components/link';
+import Button from '../../components/button';
+import { UpgradeBadge } from '../../components/badge';
+import { ITEM_TYPE_WEBLINK } from '../../common/constants';
+import Tooltip from '../../components/tooltip';
+import { CollaboratorAvatars, CollaboratorList } from '../collaborator-avatars';
+
+import InviteePermissionsMenu from './InviteePermissionsMenu';
+import messages from './messages';
+import SharedLinkSection from './SharedLinkSection';
+import EmailForm from './EmailForm';
+import getDefaultPermissionLevel from './utils/defaultPermissionLevel';
+import mergeContacts from './utils/mergeContacts';
+import type { contactType as Contact, USFProps } from './flowTypes';
+import type { SelectOptionProp } from '../../components/select-field/props';
+
+import './UnifiedShareModal.scss';
```

Comment:
nit: should this be UnifiedShareForm.scss now?

Reply:
I'll remove this import, since `UnifiedShareForm` inherits the import from `UnifiedShareModal`